### PR TITLE
Updated translations

### PR DESCRIPTION
**Pull request by splitt3r**
Updated german translations and moved some texts to the translation files. @lcharette could you also update the french files? @abdullahseba and you the arab ones?

### CODE REVIEW

**Comment by abdullahseba** [comment]
Will do. What do I add? not use to github.


________________________________

**Comment by splitt3r** [comment]
In `app/sprinkles/account/locale/de_DE/messages.php`:

- SESSION_COMPROMISED.TEXT (SESSION_COMPROMISED is an array now)
- CREATE
- CREATE.PAGE
- CREATE.SET
- SIGN_IN_HERE

In `app/sprinkles/admin/locale/en_US/messages.php`:

- NO_FEATURES_YET

In `app/sprinkles/core/locale/en_US/messages.php`:

- PAGINATION.NEXT
- PAGINATION.PREVIOUS
- PAGINATION.FIRST
- PAGINATION.LAST
- LEGAL.DESCRIPTION (LEGALis an array now)
- PRIVACY.DESCRIPTION (PRIVACYis an array now)
- WELCOME_TO

That should by all 😀

**Comment by abdullahseba** [comment]
Thanks! will try to do it soon.

**Comment by alexweissman** [comment]
Thanks!  This can be merged in a hotfix.

File: app/sprinkles/account/templates/pages/set-password.html.twig [hunk 1]
```diff
@@ -17,8 +17,8 @@
     <!-- /.login-logo -->
 
     <div class="login-box-body login-form">
-        <p class="login-box-msg"><strong>Create Password</strong></p>
-        <p class="login-box-msg">Welcome to {{site.title}}!  Please create a password for your new account.</p>
+        <p class="login-box-msg"><strong>{{translate("PASSWORD.CREATE")}}</strong></p>
+        <p class="login-box-msg">{{translate("WELCOME_TO", {'title': site.title})}} {{translate("PASSWORD.CREATE.PAGE")}}</p>
```
Comment (alexweissman) [inline app/sprinkles/account/templates/pages/set-password.html.twig]: Would this be a good spot to use [compound i18n keys](https://learn.userfrosting.com/building-pages/i18n#Theplaceholder)?  `PASSWORD.CREATE.PAGE` could reference the `WELCOME_TO` key.

Comment (splitt3r) [inline app/sprinkles/account/templates/pages/set-password.html.twig]: Sounds good :) will try that

File: app/sprinkles/admin/templates/pages/dashboard.html.twig [hunk 2]
```diff
@@ -245,9 +245,13 @@
...
                 <div class="box-footer">
-                    <h4>Welcome to {{site.title}}!</h4>
+                    <h4>
+                        {{translate("WELCOME_TO", {
+                            'title': site.title
+                        })}}
+                    </h4>
                     <p>
-                        It doesn't look like any features have been set up for this account...yet.  Maybe they haven't been implemented yet, or maybe someone forgot to give you access.  Either way, we're glad to have you aboard!
+                        {{translate("NO_FEATURES_YET")}}
```
Comment (lcharette) [inline app/sprinkles/admin/templates/pages/dashboard.html.twig]: Not sure this need translating, at least not in core sprinkles where the key will stay forever. This sentence will be changed by devs in 95% of cases maybe...

**Comment by abdullahseba** [comment]
Updated mine (I think) hope I did it right....

File: app/sprinkles/account/templates/pages/register.html.twig [hunk 3]
```diff
@@ -82,7 +82,7 @@
                 <button type="submit" class="btn btn-block btn-primary">{{translate('REGISTER_ME')}}</button>
             </div>
             <div style="padding-top: 10px;">
-                Already have an account? <a href="{{site.uri.public}}/account/sign-in">Sign in here.</a>
+                {{translate('SIGN_IN_HERE', {'url' : '#{site.uri.public}/account/sign-in'}) | raw}}
```
Comment (alexweissman) [inline app/sprinkles/account/templates/pages/register.html.twig]: This doesn't seem to work properly.

**Pull request merged**